docs/docs/cmd/teams/meeting/meeting-get.md

```diff
@@ -0,0 +1,150 @@
+# teams meeting get
+
+Get specified meeting details
+
+## Usage
+
+```sh
+m365 teams meeting get [options]
+```
+
+## Options
+
+`-u, --userId [userId]`
+: The id of the user, omit to retrieve meetings for current signed in user. Use either `id`, `userName`, or `email`, but not multiple.
+
+`-n, --userName [userName]`
+: The name of the user, omit to retrieve meetings for current signed in user. Use either `id`, `userName`, or `email`, but not multiple.
+
+`--email [email]`
+: The email of the user, omit to retrieve meetings for current signed in user. Use either `id`, `userName`, or `email`, but not multiple.
+
+`-j, --joinUrl <joinUrl>`
+: The join URL from a calendar invite.
+
+--8<-- "docs/cmd/_global.md"
+
```

Comment:
Could we add a remarks section and add an important message that the user should add an access policy when using app only permissions?
https://github.com/pnp/cli-microsoft365/issues/3900#issuecomment-1310889649